.github/workflows/test-github-backport-active.yml

```diff
@@ -0,0 +1,62 @@
+name: test-github-backport-active
+
+on:
+  merge_group: ~
+  workflow_dispatch:
+    inputs:
+      dry-run:
+        description: 'Dry run'
+        required: true
+        default: true
+        type: boolean
+  push:
+    paths:
+      - '.github/workflows/test-github-backport-active.yml'
+      - 'github/backport-active/**'
+
+permissions:
+  pull-requests: write
+  contents: read
+
+jobs:
+  test-backport-action:
```

Comment:
The tests e2e are quite opinionated.

They need a mandatory job name called `test`. Then you can run as many assertions as youneed in other jobs.

See https://github.com/elastic/oblt-actions/pull/255/files#diff-62ae283b6ef30504d453ea2409d66bacc1281a63d1e9ef77ec690ae457511275

And then you also need to include the filter in `.github/workflows/no-test.yml`:
- See https://github.com/elastic/oblt-actions/pull/255/files#diff-f0f02d8897ac2367b79589b546fd95c404c95999e2177c9506fec113aa0a0cfc